frontend/packages/console-shared/src/components/spotlight/InteractiveSpotlight.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import { PopperOptions } from 'popper.js';
+import { Popper } from '../popper';
+import './spotlight.scss';
+
+type InteractiveSpotlightProps = {
+  element: Element;
+};
+
+const InteractiveSpotlight: React.FC<InteractiveSpotlightProps> = ({ element }) => {
+  const { height, width } = element.getBoundingClientRect();
+  const style: React.CSSProperties = {
+    height,
+    width,
+  };
+  const popperOptions: PopperOptions = {
+    modifiers: {
+      preventOverflow: {
+        enabled: false,
+      },
+    },
+  };
```

Comment:
Make this a const outside of the render function.

Reply:
done